test/test_epoll.cpp

```diff
@@ -99,6 +121,39 @@ TEST(CEPoll, WaitEmptyCall)
     EXPECT_EQ(srt_cleanup(), 0);
 }
 
+TEST(CEPoll, WaitEmptyCall2)
+{
+    ASSERT_EQ(srt_startup(), 0);
+
+    SRTSOCKET client_sock = srt_socket(AF_INET, SOCK_DGRAM, 0);
+    ASSERT_NE(client_sock, SRT_ERROR);
+
+    const int yes = 1;
+    const int no = 0;
+    ASSERT_NE(srt_setsockopt(client_sock, 0, SRTO_RCVSYN, &no, sizeof no), SRT_ERROR); // for async connect
+    ASSERT_NE(srt_setsockopt(client_sock, 0, SRTO_SNDSYN, &no, sizeof no), SRT_ERROR); // for async connect
+    ASSERT_NE(srt_setsockflag(client_sock, SRTO_SENDER, &yes, sizeof yes), SRT_ERROR);
+    ASSERT_NE(srt_setsockopt(client_sock, 0, SRTO_TSBPDMODE, &yes, sizeof yes), SRT_ERROR);
```

Comment:
Again, although it was me who wrote it, now I think setting those socket options does not make sense here. Furthermore, they pollute the code.

Reply:
Good point.